lib/messagebird.js

```diff
@@ -395,6 +395,308 @@ module.exports = function (accessKey, timeout) {
           httpRequest('POST', '/lookup/' + phoneNumber + '/hlr', params, callback);
         }
       }
+    },
+
+    contacts: {
+
+      /**
+       * Create a new contact. Params is optional.
+       *
+       * @param {String} phoneNumber
+       * @param {Object} params
+       * @param {Function} callback
+       * @return void
+       */
+      create: function (phoneNumber, params, callback) {
+        if (typeof params === 'function') {
+          callback = params;
+          params = {};
+        }
+
+        params.msisdn = phoneNumber;
+
+        httpRequest('POST', '/contacts', params, callback);
+      },
+
+      /**
+       * Deletes an existing contact. The callback is invoked with an error if
+       * applicable, but the data will never contain anything meaningful as the
+       * API returns an empty response for successful deletes.
```

Comment:
We could wrap the callback in a function and return something meaningful, if you think it could be useful. What do you think?

Reply:
I've considered that, but don't think there's anything meaningful to return: if error is null, it means the resource was deleted successfully. We could return a bool to indicate success, but that's already something that can be inferred from the error - so I'm really not sure! Do you have any thoughts on that?